jac-uk/admin
# Support self-assessment extraction on Admin

## What's included?
Added ability for admin to upload/replace self assessment file and extract the content.
Added flag to application store to track status of extraction.

Closes #2411 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- View the following application in Admin: https://jac-admin-develop--pr2417-feature-2411-support-5ws5j0mb.web.app/exercise/Vxit8ipY8Xij340SIdrJ/applications/applied/application/L6JeIMFEdxortJAYyKOo
- Replace the uploaded self assessement file with another one
- The file should upload ok and a spinner should display whilst the content is being extracted
- Once the content is extracted it should be displayed in editable text input fields
- The word limit should be displayed next to each field (defaults to 250 words)
- Modify one of the fields until the word limit is exceeded. The save button should become disabled and a warning should appear
- Modify one of the other fields within the word limit and press save. The content should save ok and reappear when the page is refreshed

https://github.com/jac-uk/admin/assets/115651787/b479c997-e79d-4b24-9574-93ed1a70f6b5


## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
